library/src-bootstrapped/dotty/DottyPredef.scala

```diff
@@ -0,0 +1,41 @@
+package dotty
+
+object DottyPredef {
+
+  @forceInline final def assert(assertion: => Boolean, message: => Any): Unit = {
+    if (!assertion)
+      assertFail(message)
+  }
+
+  @forceInline final def assert(assertion: => Boolean): Unit = {
+    if (!assertion)
+      assertFail()
+  }
+
+  def assertFail(): Unit = throw new java.lang.AssertionError("assertion failed")
```

Comment:
That change would prevent bootstrapping code to use `valueOf`. I believe it's better to leave `Predef` where it was.